### DISCUSSION TITLE
Github page with custom domain is "not secure"

### DISCUSSION THREAD

**Discussion by madsravn** [community/community#23599] [Other Feature Feedback, Questions, & Ideas]
<p>Hi friends,</p>
<p>I am having problems getting my github page to use https.</p>
<p>My website is <code>www.madsravn</code>. I have attached a screenshot of my DNS settings.<br>
My github page repository is here: <a href="https://github.com/madsravn/madsravn.github.io" rel="nofollow noopener">https://github.com/madsravn/madsravn.github.io</a></p>
<p>Can anyone help me to find out what is wrong? I have activated HTTPS on my github page repo and <a href="https://madsravn.dk" rel="nofollow noopener">https://madsravn.dk</a> keeps being “not secure” in chrome at least.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/6/6f891e7ca1b39d9ce6efab25cfc743b491ebc98d.png" data-download-href="/uploads/short-url/fUGM54C1UDVpIKjQ8uKWDtSDpN3.png?dl=1" title="Skærmbillede 2020-06-02 kl. 08.15.07" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/399452/181078148-12a1fbc9-99ec-4513-8453-ee2b0e638077.png" alt="Skærmbillede 2020-06-02 kl. 08.15.07" data-base62-sha1="fUGM54C1UDVpIKjQ8uKWDtSDpN3" width="690" height="475" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/6/6f891e7ca1b39d9ce6efab25cfc743b491ebc98d_2_690x475.png, https://aws1.discourse-cdn.com/github/optimized/2X/6/6f891e7ca1b39d9ce6efab25cfc743b491ebc98d_2_1035x712.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/6/6f891e7ca1b39d9ce6efab25cfc743b491ebc98d.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/6/6f891e7ca1b39d9ce6efab25cfc743b491ebc98d_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Skærmbillede 2020-06-02 kl. 08.15.07</span><span class="informations">1044×720 73.5 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by I-Cat** [comment]
<p>I added a pull request. It shows TLs 1.2 but let’s see if I fixed it</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>Much like <a href="https://github.community/t/problem-with-fonts/116218/4">my answer in another topic</a>, you’re getting these security errors because you’re displaying content on an HTTPS webpage from HTTP sources. If you want your webpage to be truly secure, you need to use HTTPS throughout. <a class="mention" href="/I-Cat">@I-Cat</a>’s PR will help, but you need to ensure that you don’t break things again by using <code>http://</code> in any new links you add to your site.</p>
<p>I hope this helps!</p>

**Comment by I-Cat** [comment]
<p>O Forgot to say Thanks</p>
